package.json

```diff
@@ -16,27 +16,27 @@
   },
   "dependencies": {
     "@dabapps/redux-requests": "^0.5.5",
-    "@types/js-cookie": "^2.0.28",
-    "@types/node": "^7.0.18",
-    "@types/underscore": "^1.8.0",
-    "axios": "^0.18.0",
+    "@types/js-cookie": "^2.2.4",
+    "@types/node": "^10.5.0",
+    "@types/underscore": "^1.9.3",
+    "axios": "^0.18.1",
     "flux-standard-action": "^1.2.0",
-    "js-cookie": "^2.1.4",
+    "js-cookie": "^2.2.1",
     "path": "^0.12.7",
-    "path-to-regexp": "^2.1.0",
+    "path-to-regexp": "^4.0.4",
```

Comment:
There'll be breaking changes. Are you sure this is okay?

Reply:
Hmm, I relayed on test and console errors which highlighted nothing.
Is there any particular way I can be sure it is not breaking anything?

Reply:
Perhaps manually using in a project?

Reply:
Go look at the releases/change log for that library, and then compare our usage to see if we're using any features that had breaking changes.

It may just be that they dropped support for old browsers or something.

Reply:
https://github.com/pillarjs/path-to-regexp/compare/v2.1.0...v3.1.0
There are crucial changes between v2 and v3, it is all about documentation and renaming.